### DISCUSSION TITLE
Domotizzare un deumidificatore low-cost Hysure B07JDHDYZ3

### DISCUSSION THREAD

**Discussion by Cupra1979** [indomus/forum#325] [Hardware & Firmware]
Ciao a tutti! Innanzitutto, complimenti per il forum...spero vivamente che possa crescere, è utilissima una community di esperti in tema domotico!

Ne approfitto subito per una domandina a bruciapelo: ho da poco acquistato un economicissimo deumidificatore Hysure B07JDHDYZ3 su Amazon, con lo scopo di abbassare l'umidità di un sottoscala/dispensa che riportava valori di umidità di 63-64%.
L'umidità della stanza è monitorata con uno "Xiaomi Mijia Mi Temperature and Humidity Monitor 2" (il modello LYWSD03MMC) gestito con HA ed Homey. Qui il grafico che mostra l'efficacia del deumidificatore da quando è stato avviato (ieri all'ora di pranzo).
![Immagine 2021-05-18 102009](https://user-images.githubusercontent.com/69384072/118617657-214afb80-b7c3-11eb-8590-6b8a58c7255d.png)

Domandona: è possibile in qualche modo domotizzarlo, in maniera che si avvii ad esempio quando l'umidità sale oltre una certa soglia e/o si fermi sotto un determinato valore? Il deumidificatore ha un solo tasto "on/off" ed è alimentato con un alimentatore mi pare da 24v (devo verificare). Ho visto i Sonoff SV, ma non so se possano fare al mio caso...
Accetto consigli, grazie mille in anticipo!!

**Comment by sky7176** [comment]
Ciao,
come è il comportamento se, una volta acceso, manca tensione?
Mi spiego meglio, fai questa prova:
1) lo accendi premento il tasto on/off
2) dopo che è partito togli la spina di alimentazione, lui si spegne
3) rimetti la spina di alimentazione, il deumidificatore riparte o devi ripremere il tasto on/off?

Se alla risposta 3 hai un "si riparte senza premere il tasto on/off" sei a cavallo, con un sonoff lo accendi e lo spegni, al contrario se al punto 3, dopo aver rimesso l'alimentazione, per farlo ripartire devi premere il tasto on/off la cosa è un po più complicata, nel senso che senza "manomettere" l'oggetto credo sia dura poterlo domotizzare.

Per manomettere intendo dire, aprire l'oggetto e mettere in parallelo al pulsante on/off il contatto "pulito" di un sonoff.
Per contatto pulito intendo un contatto che simuli il pulsante, il sonoff originale non lo fa, riporta sui contatti del rele la tensione a 230 V ac.
Occhio che con la tensione non si scherza, ci si fa male se non si sa cosa si stà facendo.
Su come modificare un sonoff per rendere il contatto di uscita pulito ci sono diverse guide, ma ripeto se non hai idea di cosa stai facendo, non farlo.

Giuseppe

**Reply by Cupra1979** [reply]
Ciao Giuseppe, grazie mille per la risposta.
Ti confermo che se viene spento per mancanza di alimentazione, è da riaccendere premendo l'unico pulsante di cui è dotato.
In effetti avevo già valutato l'utilizzo di una semplice smart plug, ma purtroppo togliendo alimentazione non riparte da solo...per questo stavo pensando a come domotizzarlo aggiungendo un Sonoff, dopo aver letto qualche guida per la modifica in "contatto pulito".
Considera che lavora in bassa tensione, infatti è alimentato da un alimentatore a 24v, per questo mi domandavo quale fosse il dispositivo più adatto a quello che voglio fare. Ho un po' di manualità in elettronica (saldature a stagno ecc..), ma senza sconfinare troppo nel lavoro dell'elettricista :)
L'ideale per il mio caso sarebbe un qualche dispositivo a bassa tensione DC, con contatto pulito (o al limite modificabile in tal senso) e idealmente ZigBee, ma va bene anche semplicemente WiFi.

**Reply by Larry-78** [reply]
Ciao @Cupra1979  ci sarebbero varie soluzioni la prima che mi viene in mente che non comporta alcuna modifica è lo Shelly 1
lo puoi alimentare a 220v o a 12v, basta cambiare la posizione di un jumper all'interno ed è munito di un contatto pulito.

**Reply by Cupra1979** [reply]
Grazie @Larry-78 per il suggerimento. Al momento ho preso 3 Sonoff SV da provare, che visto il costo (3 pz. a 16€ spediti) possono essere una buona "palestra" per un quasi neofita. Da quel che ho capito possono essere utili per domotizzare un po' di tutto purché sia a bassa tensione...il mio obiettivo è quello di poterli integrare con HA ed Homey e vedo che per entrambi gli hub esiste un'integrazione apposita.

**Reply by Larry-78** [reply]
..di nulla...effettivamente ci sono varie soluzioni per tutte le tasche ;-)

**Reply by sky7176** [reply]
Ciao potresti usare un esp01 relay lo trovi sui soliti siti cinesi.
Ho il contatto pulito lo tasmotizzi e il gioco è fatto. 
Per alimentazione la prelevi dal 24 vcc e la porti a 5vcc anche se credo cge ci sia la versione a 24 vcc del esp01

**Reply by msillano** [reply]
Sono d'accordo a proposito di Sonoff, è un oggetto meraviglioso, con un rapporto qualità prezzo imbattibile e l'ho usato in molti progetti ([PDMtimer](https://github.com/msillano/Ozone-coronavirus-sonoff/tree/master/PROJECTS-DIY/timerPDM), [ebb-and-flow](ebb-and-flow)...).

Tuttavia non lo userei in questa applicazione. La modifica per un 'contatto pulito' non è poi così banale e priva di rischi.

Cerca piuttosto un dispositivo con un contatto pulito nativo, a 220 V o 5 V. Ad esempio Io ho usato questo [switch-1CH](tuyaDAEMON/devices/switch-1CH/device_switch-1CH.pdf), credo sia possibile " tasmotizzarlo", se vuoi usarlo in questo modo.

Buon lavoro.
m.s.
